pom.xml

```diff
@@ -30,6 +30,7 @@
     <gitHubRepo>jenkinsci/bitbucket-branch-source-plugin</gitHubRepo>
     <jenkins.version>2.401.3</jenkins.version>
     <hpi.compatibleSinceVersion>2.0</hpi.compatibleSinceVersion>
+    <useBeta>true</useBeta> <!-- Jenkins.MANAGE -->
```

Comment:
Is it supposed to stay or was it to test ?

Reply:
OK, I read badly this conf, I thought it was forcing the plugin in beta mode -_-
My bad, sorry.

Reply:
no worries, i've also started [JENKINS-73089](https://issues.jenkins.io/browse/JENKINS-73089) to address moving the `Overall/Manage` permission out of `beta` status